cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -19,15 +19,17 @@
                 <% if (templates[i].tab == "common") { %>
                     <% if (!templates[i].boilerplate_name) { %>
                     <li class="editor-md empty">
-                        <button type="button" class="button-component" data-category="<%= templates[i].category %>">
-                            <span class="name"><%= templates[i].display_name %></span>
+                        <button type="button" class="button-component" data-category="<%- templates[i].category %>">
+                            <%= HtmlUtils.HTML(support_indicator_template({support_level: templates[i].support_level})) %>
```

Comment:
No-- that would escape the template so that users would visually see the HTML tags. You use <%- for plain text that should be escaped.

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#underscore-js-template-files

If you need to pass an HTML snippet to a template, which has already been HTML-escaped, you should name the variable with an Html suffix, and use HtmlUtils.ensureHtml() to ensure that it was in fact properly HTML- escaped. See the following example.

````
<%= HtmlUtils.ensureHtml(nameHtml) %>